virl2_client/event_handling.py

```diff
@@ -141,23 +228,6 @@ def parse_event(self, event: Event) -> None:
                     event.element = event.lab.get_link_by_id(event.element_id)
             except ElementNotFound:
                 return
```

Comment:
You're not logging these because it's common that an event arrives for a recently-deleted element?

Reply:
Yep. When a node is deleted, the node deletion event arrives first, which locally deletes everything related to that node, so the slew of interface/link deletion events afterwards are useless, for example.